lib/fabrik/VirtualHostManager.js

```diff
@@ -0,0 +1,168 @@
+const _ = require('lodash');
+const Promise = require('bluebird');
+const bosh = require('../bosh');
+const utils = require('../utils');
+const VirtualHostInstance = require('./VirtualHostInstance');
+const BaseManager = require('./BaseManager');
+const VirtualHostAgent = require('./VirtualHostAgent');
+const logger = require('../logger');
+const errors = require('../errors');
+const NotImplemented = errors.NotImplemented;
+const ServiceInstanceNotOperational = errors.ServiceInstanceNotOperational;
+const BadRequest = errors.BadRequest;
+const ServiceBindingAlreadyExists = errors.ServiceBindingAlreadyExists;
+const ServiceBindingNotFound = errors.ServiceBindingNotFound;
+const NotFound = errors.NotFound;
+const mapper = require('./VirtualHostRelationMapper');
+
+class VirtualHostManager extends BaseManager {
+
+  constructor(plan) {
+    super(plan);
+    this.director = bosh.director;
+    this.agent = new VirtualHostAgent(this.settings.agent);
+    this.mapper = mapper.VirtualHostRelationMapper;
+  }
+
+  verifyFeatureSupport(feature) {
+    if (!_.includes(this.agent.features, feature)) {
+      throw new NotImplemented(`Feature '${feature}' not supported`);
+    }
+  }
+
+  getDeploymentManifest(deploymentName) {
```

Comment:
Once the [below](https://github.com/cloudfoundry-incubator/service-fabrik-broker/pull/111/files#r157011348) comment is incorporated, this method can be removed.